core/src/com/unciv/ui/utils/extensions/Scene2dExtensions.kt

```diff
@@ -313,7 +313,7 @@ fun Actor.surroundWithCircle(size: Float, resizeActor: Boolean = true, color: Co
     return IconCircleGroup(size, this, resizeActor, color)
 }
 
-fun Actor.surroundWithThinCircle(color: Color=Color.BLACK): IconCircleGroup = surroundWithCircle(width*1.05f, false, color)
+fun Actor.surroundWithThinCircle(color: Color=Color.BLACK): IconCircleGroup = surroundWithCircle(width+2f, false, color)
```

Comment:
This we might need to pay around with to make sure it looks good for different sizes of circles, but we can deal with that

Reply:
👍🏿